tests/python/eeg_metrics.py

```diff
@@ -38,9 +39,14 @@ def main ():
     params.ip_protocol = args.ip_protocol
     params.timeout = args.timeout
     params.file = args.file
-
-    board = BoardShim (args.board_id, params)
-    sampling_rate = BoardShim.get_sampling_rate (args.board_id)
```

Comment:
and the same here, lets move to another file. I really want to keep code samples from docs less than one screen of code